frontend/test/__support__/e2e/helpers/e2e-bi-basics-helpers.js

```diff
@@ -0,0 +1,52 @@
+export function summarize({ mode } = {}) {
```

Comment:
The idea for the future is that we can expand this with metrics and columns, so that we can hide the implementation details even further and simplify test even more.

For example, think of it like this:
```js
summarize({
      mode: "notebook",
      metric: "Average of ...",
      field: "Quantity",
    });
```
The next iteration could cover adding multiple metrics or multiple columns:
```js
summarize({
      mode: "notebook",
      metric: "Average of ...",
      field: ["Quantity", "Total"],
    });
```

Because we have at least two different modes (simple and notebook), and because even notebook mode doesn't always look the same, there are a lot of edge cases that we need to cover here, but the main idea still stands.